subprojects/logging/src/integTest/groovy/org/gradle/DeprecationHandlingIntegrationTest.groovy

```diff
@@ -137,7 +137,7 @@ class DeprecationHandlingIntegrationTest extends AbstractIntegrationSpec {
         given:
         def initScript = file("init.gradle") << """
             allprojects {
-                DeprecationLogger.nagUserOfPluginReplacedWithExternalOne("DeprecatedPlugin", "Foobar") // line 2
+                org.gradle.util.DeprecationLogger.nagUserOfPluginReplacedWithExternalOne("DeprecatedPlugin", "Foobar") // line 2
```

Comment:
:o: this may indicate that we'll need a public API for showing deprecation warnings in the future (like, plugin authors themselves wanting to deprecate stuff).